kustomize/commands/build/flagloadrestrictor.go

```diff
@@ -23,6 +24,15 @@ func AddFlagLoadRestrictor(set *pflag.FlagSet) {
 			"relocatability of the kustomization.")
 }
 
+func AddFlagLoadRestrictorCompletion(cmd *cobra.Command) {
+	cmd.RegisterFlagCompletionFunc(flagLoadRestrictorName, func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
This is currently not passing lint, because the error return is not being treated. Could you please bubble up the error returned here and handle it in the `NewCmdBuild` function?

Reply:
@sylr hi, can you address the comment here so that we can process the PR? Thanks!